plasmapy/formulary/collisions/helio/collisional_analysis.py

```diff
@@ -0,0 +1,360 @@
+"""
+Module containing the Collisional Analysis formulation.
+"""
+__all__ = ["thermalization_ratio"]
+
+import astropy.units as u
+import numbers
+import numpy as np
+
+from plasmapy.particles import Particle, ParticleLike, ParticleList
+from plasmapy.utils.decorators import validate_quantities
+
+
+@validate_quantities(
+    T_1={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    T_2={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+)
+def thermalization_ratio(
+    *,
+    r_0: u.au,
+    r_n: u.au,
+    n_1: u.cm**-3,
+    n_2: u.cm**-3,
+    v_1: u.km / u.s,
+    T_1: u.K,
+    T_2: u.K,
+    ions: ParticleLike = [Particle("p+"), Particle("He-4++")],
+    n_step: int = 100,
+    density_scale: float = -1.8,
+    velocity_scale: float = -0.2,
+    temperature_scale: float = -0.74,
+):
+    r"""
+    Contains the functionality to calculate the collisional
+    thermalization a plasma will undergo in transit, taken from
+    :cite:t:`maruca:2013`. Coulomb collisions are soft, small angle
+    deflections mediated by the electrostatic force and occur between
+    constituent particles of a plasma :cite:t:`baumjohann:1997`.
+    These collisions cause the plasma to thermalize over distance,
+    i.e. the temperature of the plasma will approach thermal
+    equilibrium. This function allows this thermalization to be
+    modeled and predicts the temperature ratio, for different ion
+    species within the plasma, at a different point in space.
```

Comment:
Perhaps start with a one or two liner statement of what the function does, and then have this in either the second paragraph or the notes section?